src/get-changed-file.ts

```diff
@@ -19,7 +19,8 @@ export async function getChangedFiles(): Promise<ChangedFiles> {
   const globs = pattern.length ? pattern.split(',') : ['**.php'];
   const isMatch = picomatch(globs);
   console.log('Filter patterns:', globs, isMatch('src/test.php'));
-  const payload = github.context.payload as Webhooks.WebhookPayloadPullRequest;
+  const payload = github.context
+    .payload as Webhooks.EventPayloads.WebhookPayloadPullRequest;
```

Comment:
I don't think this is necessary, it's only breaking the line, right?

Reply:
The EventPayloads is added (which after checking all the commits was the way how the build was fixed)

Reply:
Ok, never mind then. I overlooked that.